tests/Avalonia.RenderTests/Media/VisualBrushTests.cs

```diff
@@ -19,18 +19,11 @@ public VisualBrushTests()
         {
         }
 
-        private string BitmapPath
-        {
-            get { return System.IO.Path.Combine(OutputPath, "github_icon.png"); }
-        }
+        private string BitmapPath => System.IO.Path.Combine(OutputPath, "github_icon.png");
 
-        private Control Visual
+        private Control Visual => new Panel
```

Comment:
If there is a code block in a getter you really shouldn't be "modernizing" it IMO. It is entirely developer preference in that case.